flocker/node/agents/cinder.py

```diff
@@ -411,6 +413,7 @@ def __init__(self,
         self.nova_volume_manager = nova_volume_manager
         self.nova_server_manager = nova_server_manager
         self.cluster_id = cluster_id
+        self.timeout=CINDER_VOLUME_DESTRUCTION_TIMEOUT
```

Comment:
1. Maybe `delete_timeout` or `_delete_timeout` for more accurate explanation of how it's used.
2. Document the attribute in the class docstring.